mammooc-rating-widget.html

```diff
@@ -153,6 +153,9 @@
         _generateRatingUrl: function(provider, courseId) {
             return "https://mammooc.org/evaluations/export_overall_course_rating?provider=" + this.provider + "&course_id=" + this.courseId;
         },
+        _generateCourseUrl: function(provider, courseId) {
+            return "https://mammooc-dev.herokuapp.com/courses/" + this.provider + "~" + this.courseId;
```

Comment:
No, it's not for the master, but it is for the dev branch. I need to push a change to the master directly after merging this PR.